did_peer/src/peer_did/transform.rs

```diff
@@ -0,0 +1,20 @@
+use crate::error::DidPeerError;
+
+#[derive(Clone, Debug, PartialEq)]
```

Comment:
Same as for `Numalgo`, this can be `Copy` and returning it would be cheap instead of passing around references.